src/request_list.js

```diff
@@ -638,7 +641,7 @@ export class RequestList {
  *
  *   If `null`, the list will not be persisted and will only be stored in memory. Process restart
  *   will then cause the list to be crawled again from the beginning. We suggest always using a name.
- * @param {Object[]|string[]} sources
+ * @param {Array.<Request|RequestOptions|string>} sources
```

Comment:
same here

Reply:
Done. And on a few other places, I missed before.